docker-compose.yml

```diff
@@ -3,19 +3,19 @@ services:
   db:
     container_name: respa-db
     build: ./docker/postgres/
-    restart: always
+    restart: on-failure
     environment:
         POSTGRES_USER: respa
         POSTGRES_PASSWORD: respa
         POSTGRES_DB: respa
     volumes:
-      - db_data:/var/lib/postgresql
+      - db_data:/var/lib/postgresql/data
   api:
     container_name: respa-api
     build:
       context: ./
       dockerfile: ./docker/dev/Dockerfile
-    command: bash -c 'tail -f /dev/null'
+    command: python manage.py runserver 0:8000
```

Comment:
Does it really work to just run the server?  README.md says to run the server with a separate command https://github.com/City-of-Helsinki/respa/blob/master/README.md#installation-with-docker .  This is now not in sync with what is said in the README.

Reply:
true, gets out of sync with the README.

Reply:
updated README 🙂